src/features/comments/commentAPI.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  createComment,
+  deleteComment,
+  getPostComments,
+} from '../../api/comments';
+import { CommentData } from '../../types/Comment';
+
+export function fetchComments(postId: number) {
+  return getPostComments(postId);
+}
+
+export function fetchCreateComments(data: CommentData) {
+  return createComment(data);
+}
+
+export function fetchDeleteComments(commentId: number) {
+  return deleteComment(commentId);
+}
```

Comment:
Why do you need this? You can directly use `getPostComments, createComment` and so on

Reply:
The same comment for all similar cases